packages/docs-ui/README.md

```diff
@@ -75,3 +76,25 @@ import { SideNavigationSection } from '@braid-design-system/docs-ui';
 
 [`HeadingLevel`]: https://seek-oss.github.io/braid-design-system/components/Heading
 [Braid Design System]: https://seek-oss.github.io/braid-design-system/
+
+### `HeaderNavigation`
+
+A navigation section containing a list of links, designed for the side bar.
+
+```tsx
+import { SideNavigationSection } from '@braid-design-system/docs-ui';
+
+<SideNavigationSection title={title} items={items} />;
+```
```

Comment:
This looks like it still needs to be updated.

Reply:
Ahh yep, my mistake